Formula/fontconfig.rb

```diff
@@ -29,6 +29,7 @@ class Fontconfig < Formula
   end
 
   depends_on "pkg-config" => :build
+  depends_on "python@3.9" => :build
```

Comment:
Should we try to use the latest python?

```suggestion
  depends_on "python@3.10" => :build
```

Reply:
Yes we should try 3.10 right away, if the build passes with that.